cmd/ddev/cmd/global_dotddev_assets/commands/host/heidisql

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Run HeidiSQL against current db
+## Usage: heidisql
+## Example: "ddev heidisql"
+## OSTypes: linux,windows
+
+arguments="--host=127.0.0.1 --port=${DDEV_HOST_DB_PORT} --user=root --password=root --description=${DDEV_SITENAME}"
+
+case $OSTYPE in
+  linux-gnu)
+    # "/mnt/c/Program Files/HeidiSQL/heidisql.exe"
+    # HeidiSQL is Microsoft only, but we want to starting it from WSL2
+    "/mnt/c/Program Files/HeidiSQL/heidisql.exe" "$arguments" &
+    echo "Attempted to launch heidisql.exe"
+    ;;
+
+  "win*"* | "msys"*)
+    # 'C:\Program Files\HeidiSQL\heidisql.exe'
+    # You may need to add it to your system %PATH% or change the path here
```

Comment:
The comment here isn't correct is it, since pathing is not used... It probably *should* be used, and remove the hard-coded path.

Reply:
I overtook the comment from the the MySQL Workbench example ;) HeidiSQL did not write itself into the path variable, so we have to use the hard coded path here 😞